rotkehlchen/tests/api/test_sushiswap.py

```diff
@@ -0,0 +1,509 @@
+import random
+import warnings as test_warnings
+from contextlib import ExitStack
+from http import HTTPStatus
+
+import pytest
+import requests
+
+from rotkehlchen.assets.asset import EthereumToken
+from rotkehlchen.chain.ethereum.manager import NodeName
+from rotkehlchen.chain.ethereum.modules.sushiswap import (
+    SushiswapPoolEvent,
+    SushiswapPoolEventsBalance,
+    SUSHISWAP_EVENTS_PREFIX,
+)
+from rotkehlchen.chain.ethereum.modules.ammswap.typing import EventType
+from rotkehlchen.chain.ethereum.trades import AMMSwap, AMMTrade
+from rotkehlchen.chain.ethereum.typing import string_to_ethereum_address
+from rotkehlchen.constants.misc import ZERO
+from rotkehlchen.fval import FVal
+from rotkehlchen.premium.premium import Premium
+from rotkehlchen.tests.utils.api import (
+    ASYNC_TASK_WAIT_TIMEOUT,
+    api_url_for,
+    assert_error_response,
+    assert_ok_async_response,
+    assert_proper_response_with_result,
+    assert_simple_ok_response,
+    wait_for_async_task,
+)
+from rotkehlchen.tests.utils.ethereum import INFURA_TEST
+from rotkehlchen.tests.utils.rotkehlchen import setup_balances
+from rotkehlchen.typing import AssetAmount, Location, Price, Timestamp, TradeType
+
+
+SWAP_ADDRESS = string_to_ethereum_address('0x63BC843b9640c4D79d6aE0105bc39F773172d121')
+
+
+@pytest.mark.parametrize('ethereum_accounts', [[SWAP_ADDRESS]])
+@pytest.mark.parametrize('start_with_valid_premium', [True])
+@pytest.mark.parametrize('ethereum_modules', [['compound']])
+def test_get_balances_module_not_activated(
+        rotkehlchen_api_server,
+        ethereum_accounts,  # pylint: disable=unused-argument
+):
+    response = requests.get(
+        api_url_for(rotkehlchen_api_server, 'sushiswapbalancesresource'),
+    )
+    print(response.text)
+    assert_error_response(
+        response=response,
+        contained_in_msg='sushiswap module is not activated',
+        status_code=HTTPStatus.CONFLICT,
+    )
+
+
+UNISWAP_TEST_OPTIONS = [
+    # Test with infura (as own node), many open nodes, and premium + graph
+    (False, INFURA_TEST, (NodeName.OWN, NodeName.MYCRYPTO)),
+    (False, '', (NodeName.MYCRYPTO, NodeName.BLOCKSCOUT, NodeName.AVADO_POOL)),
+    (True, '', ()),
+]
+# Skipped infura and many open nodes for now in the CI. Fails flakily due to timeouts
+# from time to time. We should run locally to make sure that it still works.
+SKIPPED_UNISWAP_TEST_OPTIONS = [UNISWAP_TEST_OPTIONS[-1]]
+
+
+@pytest.mark.parametrize('ethereum_accounts', [[SWAP_ADDRESS]])
+@pytest.mark.parametrize('ethereum_modules', [['uniswap']])
+@pytest.mark.parametrize(
+    'start_with_valid_premium,ethrpc_endpoint,ethereum_manager_connect_at_start',
+    SKIPPED_UNISWAP_TEST_OPTIONS,
+)
+def test_get_balances(
+        rotkehlchen_api_server,
+        ethereum_accounts,  # pylint: disable=unused-argument
+        rotki_premium_credentials,
+        start_with_valid_premium,
+):
+    """Check querying the uniswap balances endpoint works. Uses real data
```

Comment:
Hmm ... man this entire test, tests uniswap. You use the uniswap module, call uniswal balance endpoint. Too much copy past :sweat_smile: 

Please change it to test sushiswap